action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Pull request summarizer'
+name: 'Etendo Pull request summarizer'
```

Comment:
GPT summary of [456961](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/50a2e65d8816904f457cb4d5390f2804a4cfd5a7/action.yml#456961b0d1e401d3acee823d274d8979f31a7015) - [fb2800](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/b0d24914146721c2c98a413653af9d3da434f9f1/action.yml#fb28002199560c3958df9080e005aba59362a3fd):
* Changed the name from 'Pull request summarizer' to 'Etendo Pull request summarizer'
* Updated the description
* Added an input parameter 'repository'